api/deployment.go

```diff
@@ -166,3 +167,15 @@ func storeSiteInfo(data CoreAppData) map[int64]SitesReponse {
 	}
 	return siteInfo
 }
+
+func CurrentPlan() (bool, error) {
```

Comment:
Here we can just return the error right, no need to check the `bool` value again